Make python_check relocatable

kchodorow opened the pull request:
Uses $location in the genrule and a path that's relative to the
script to find expected files so that this works both as a local
rule and if tensorflow is used as a remote repository.

tensorflow-jenkins commented:
Can one of the admins verify this patch?

ebrevdo commented on util/python/python_config.sh:
```diff
@@ -16,7 +16,11 @@
+# TODO: remove once runfiles paths are fixed.
+script_path=$(dirname $(dirname $(dirname "$0")))
+script_path=${script_path:-.}
+EXPECTED_PATHS="$script_path/util/python/python_include $script_path/util/python/python_lib $script_path/third_party/py/numpy/numpy_include"
```
nit: can you make this < 80 columns?

kchodorow replied:
Done.

ebrevdo commented on util/python/python_config.sh:
```diff
@@ -16,7 +16,11 @@
 
-EXPECTED_PATHS="util/python/python_include util/python/python_lib third_party/py/numpy/numpy_include"
+# Prefix expected paths with ./ locally and external/reponame/ for remote repos.
+# TODO: remove once runfiles paths are fixed.
```
Can you add yourself to the TODO?  maybe link to a github issue for the runfile paths?

kchodorow replied:
Done.

ebrevdo commented:
Thanks!  Minor comments.

kchodorow committed:
Make python_check relocatable

Uses $location in the genrule and a path that's relative to the
script to find expected files so that this works both as a local
rule and if tensorflow is used as a remote repository.

martinwicke commented:
Jenkins, test this please.

kchodorow commented:
How do I run it on Jenkins?

ebrevdo commented:
Can you rebase on master & squash commits?

vrv commented:
@ebrevdo: there's only one commit, and the branch has no conflicts with master.  We can probably just accept as is.

martinwicke commented:
Don't worry, our script can do that.

martinwicke commented:
merged.
